setup.cfg

```diff
@@ -31,13 +31,14 @@ packages = find:
 include_package_data = True
 zip_safe = False
 install_requires =
-    numpy>=1.16.6
-    scipy>=1.0
+    numpy>=1.22
+    scipy>=1.8,<1.12
     packaging
 setup_requires =
-    numpy>=1.13.3
-    scipy>=1.0
-    cython>=0.29.20
+    numpy>=1.19
+    scipy>=1.8
```

Comment:
It seems a bit odd to allow `scipy>1.12` for building the package but not for installing it, so just double-checking this.

Reply:
We had a few issues when 1.12 got out.
While this is probably fixed, our tests are all done with this respected. I think we should leave it as is for beta 1 and remove it for the next release

Reply:
Sounds good.